docs/citation.rst

```diff
@@ -0,0 +1,20 @@
+Citation
+==========
+
+How to cite Uxarray
+-----------------------
+
+Cite Uxarray using the following text:
+
+**UXARRAY. (Year).
```

Comment:
Just a little thing here: Add "Organization" to the beginning after the word "UXARRAY".